priv/templates/enum.ex.eex

```diff
@@ -1,5 +1,5 @@
 defmodule <%= @module %> do
-  @moduledoc false
+  @moduledoc <%= inspect(@module_doc) %>
```

Comment:
Choose to keep it as a simple string so it can be easily extended in the future without modifying the `.eex` templates.

Reply:
I would prefer if we don't add `@moduledoc` when we want to include docs. `@moduledoc nil` isn't normally used in Elixir code.

Reply:
@ericmj you are right! I pushed some commits addressing your comments. Thank you for the quick review!